tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,64 @@
+import os
+from threading import Thread
+import pytest
+from selenium import webdriver
+import requests
+from todo_app.app import create_app
+
+def requestJson (method, url, args):
+    r = requests.request(method, url, params=args) 
+    r.raise_for_status()
+    print(r.raise_for_status())
+    return r.json()
+    
+    
+#Function to create trello board
+def createTrelloBoard(boardName):
+    url='https://api.trello.com/1/boards/'
+    params={
+            'key': os.getenv('TRELLO_KEY'),
+            'token': os.getenv('TRELLO_TOKEN'),
+            'name': boardName
+        }
+    
+    res=requestJson("POST",url, params)
+    return res
+
+
+#Function to remove trello board
+def deleteTrelloBoard(boardId):
+    url='https://api.trello.com/1/boards/'+boardId
+    res=requestJson("DELETE",url, {})
+    return res
+
+@pytest.fixture(scope='module')
+def app_with_temp_board():
+    # Create the new board & update the board id environment variable
+    board_id = (createTrelloBoard('Selenium_Test_Board'))["id"]
```

Comment:
Note that this requires your environment variables in order to talk to Trello, which aren't set up with Pytest by default. Instead you will need to import the dotenv functions as with your integration test and add a couple of lines before this to find & load your `.env` file (rather than `.env.test`) so that the test has the right credentials to talk to Trello, i.e.:
```python
    file_path = find_dotenv('.env')
    load_dotenv(file_path, override=True)
```